Sources/Algorithm.swift

```diff
@@ -44,7 +44,7 @@ public extension StagedChangeset where Collection: RangeReplaceableCollection, C
     ///
     /// - Complexity: O(n)
     @inlinable
-    public init(source: Collection, target: Collection, section: Int) {
+    init(source: Collection, target: Collection, section: Int) {
```

Comment:
Has redundant access levels become warned by Xcode?

Reply:
Xcode 10.2 now throws this warning: "'public' modifier is redundant for initializer declared in a public extension"